VotingApplication/VotingApplication.Web.Api/Partials/Poll.html

```diff
@@ -63,6 +63,9 @@
                     <div class="col-md-1"></div>
                     <div class="col-md-10 centered">
                         <div id="votingArea"></div>
+                        <div class="text-center">
+                            <button class="btn btn-danger" data-bind="click: $root.clearVote">Delete Vote</button>
+                        </div>
```

Comment:
This displaces the ranked vote button rather than sitting next to it, so the ranked poll's "Submit vote" button is off to the left.